packages/p2p-v2/src/components/Modals/OrderDetailsConfirmModal/OrderDetailsConfirmModal.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { FileUploaderComponent } from '@/components/FileUploaderComponent';
+import { getErrorMessage, maxPotFileSize, TFile } from '@/utils';
+import { Button, InlineMessage, Modal, Text, useDevice } from '@deriv-com/ui';
+import './OrderDetailsConfirmModal.scss';
+
+type TOrderDetailsConfirmModalProps = {
+    isModalOpen: boolean;
+    onRequestClose: () => void;
+};
+
+type TDocumentFile = {
+    errorMessage: string | null;
+    files: TFile[];
+};
+
+const OrderDetailsConfirmModal = ({ isModalOpen, onRequestClose }: TOrderDetailsConfirmModalProps) => {
+    const [documentFile, setDocumentFile] = useState<TDocumentFile>({ errorMessage: null, files: [] });
+    const { isMobile } = useDevice();
+    const buttonTextSize = isMobile ? 'md' : 'sm';
+
+    const handleAcceptedFiles = (files: TFile[]) => {
+        if (files.length > 0) {
+            setDocumentFile({ errorMessage: null, files });
+        }
+    };
+
+    const removeFile = () => {
+        setDocumentFile({ errorMessage: null, files: [] });
+    };
+
+    const handleRejectedFiles = (files: TFile[]) => {
+        setDocumentFile({ errorMessage: getErrorMessage(files), files });
+    };
+
+    // TODO: uncomment this when implementing the OrderDetailsConfirmModal
+    // const displayPaymentAmount = removeTrailingZeros(
+    //     formatMoney(local_currency, amount_display * Number(roundOffDecimal(rate, setDecimalPlaces(rate, 6))), true)
+    // );
+
+    return (
+        <Modal
+            ariaHideApp={false}
+            className='p2p-v2-order-details-confirm-modal'
+            isOpen={isModalOpen}
+            onRequestClose={onRequestClose}
+        >
+            <Modal.Header
+                className='border-none lg:py-14 lg:px-[2.4rem] py-4 px-[1.6rem]'
+                onRequestClose={onRequestClose}
+            >
+                <Text size='md' weight='bold'>
+                    Payment confirmation
+                </Text>
+            </Modal.Header>
+            <Modal.Body className='flex flex-col lg:px-[2.4rem] px-[1.6rem]'>
+                <Text size='sm'>
+                    Please make sure that you’ve paid 9.99 IDR to client CR90000012, and upload the receipt as proof of
```

Comment:
why do we have fixed amount here?

Reply:
for now I have left it as an example, cause in the future we will replace the values with the proper values later 🙏

Reply:
i think we can add a TODO here as well, or in the above line, for now we can give a dummy value to displayPaymentAmount and use it here, so that its not missed out.

Reply:
hmmm we shouldn't really flood the whole app with TODO's unless its absolutely necessary like removing the FileDropzone.